### ISSUE TITLE
ntfy.sh username + password authentication not working

### ISSUE THREAD

**Issue by etfz** [louislam/uptime-kuma#3696]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### Description

The notification test feature works, but not actual monitor events.

### 👟 Reproduction steps

- Set up ntfy.sh notification with username and password authentication
- Trigger notification event

### 👀 Expected behavior

Notification sent.

### 😓 Actual Behavior

Notification not sent.

### 🐻 Uptime-Kuma Version

1.23.0

### 💻 Operating System and Arch

Debian 11.6

### 🌐 Browser

Firefox

### 🐋 Docker Version

20.10.5

### 📝 Relevant log output

```shell
Error: Error: AxiosError: Request failed with status code 403 {"code":40301,"http":403,"error":"forbidden","link":"https://ntfy.sh/docs/publish/#authentication"}
    at Ntfy.throwGeneralAxiosError (/app/server/notification-providers/notification-provider.js:38:15)
    at Ntfy.send (/app/server/notification-providers/ntfy.js:72:18)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Function.sendNotification (/app/server/model/monitor.js:1367:21)
    at async beat (/app/server/model/monitor.js:917:21)
    at async safeBeat (/app/server/model/monitor.js:985:17)
```

**Comment by CommanderStorm** [comment]
According to https://ntfy.sh/docs/publish/#authentication I don't see a problem on our side.


https://github.com/louislam/uptime-kuma/blob/62f443471163a52b7ae5ef3a30ddf9f3776e6958/server/notification-providers/ntfy.js#L16-L19

given that I have not set up a custom sever instace, could you make sure that you have made no typos in the `username`+`password` (whitespace, encoding, ...)

**Comment by etfz** [comment]
It might have been some corruption from an earlier version or something, then. The test notification always worked, but I never seemed to receive any notifications on actual events. Only the other day I checked the logs and noticed the authentication error. Creating a new notification type with the same credentials worked, and I have now switched the token authentication, which was not available when I first set this up. Just going to close this.
